### DISCUSSION TITLE
In Editor, When making a slider, pressing 1,2, or 4 should place the slider, not delete it.

### DISCUSSION THREAD

**Discussion by Xinnoh** [ppy/osu#23461] [Ideas] [answered]
Switching to select, circle, or spinner deletes the slider you're making, doesn't match stable.

When mapping on laptop trackpads, clicking becomes draining over time (esp macbooks). Using the keyboard to place sliders reduces rsi

https://github.com/ppy/osu/assets/30382015/75d2c9e6-50e5-41e0-9c5b-740f40523e2c

**Comment by Xinnoh** [comment]
Should also add that pressing esc should delete the slider, and not instantly exit the editor

https://github.com/ppy/osu/assets/30382015/725b8629-3cd6-41a6-8a22-195e76e36afb

**Comment by peppy** [comment]
I think this is valid as most stable mappers are used to this behaviour.

In addition, I'll add the "select" binding (defaults to enter) to also confirm placement, as that seems like a good UX to have.

**Reply by peppy** [reply, marked as answer]
#23508 #23509
